NDcPP_v2_2e.adoc

```diff
@@ -4169,19 +4226,41 @@ FIA_X509_EXT.1 X.509 Certificate Validation
 FIA_X509_EXT.2 X.509 Certificate Authentication
 ____
 
-*FCS_TLSS_EXT.1.1* The TSF shall implement [selection: _TLS 1.2 (RFC 5246), TLS 1.1 (RFC 4346)_] and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
+*FCS_TLSS_EXT.1.1* The TSF shall implement [selection: _TLS 1.3 (RFC 8446), TLS 1.2 (RFC 5246)_] and [selection: _TLS 1.1 (RFC 4346, no additional TLS version)_] and reject all other TLS and SSL versions. The TLS implementation will support the following ciphersuites:
 
 * {blank}
 +
 ____
-_[assignment: list of optional ciphersuites and reference to RFC in which each is defined]_ and no other ciphersuites.
+_[selection: 
+
+* _select supported ciphersuites for TLSv1.1 and/or TLSv1.2 from List 1_
+* _select supported ciphersuites for TLSv1.3 from List 2_
+
+]_ and no other ciphersuites. 
```

Comment:
@qtrinh3 Check italics, add _ after selection: in line 4234 and _ before ] in line 4239